src/openrct2/actions/NetworkModifyGroupAction.cpp

```diff
@@ -22,6 +22,15 @@ NetworkModifyGroupAction::NetworkModifyGroupAction(
 {
 }
 
+void NetworkModifyGroupAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("type", _type);
+    visitor.Visit("groupId", _groupId);
```

Comment:
Do we even expose the group id to be able to call this action? Again not sure if this has much use to a plugin author.

Reply:
Sure:
```
    interface PlayerGroup {
        readonly id: number;
        name: string;
        permissions: PermissionType[];
    }
```